### DISCUSSION TITLE
ใส่ GitHub Webhook ใน Discord

### DISCUSSION THREAD

**Discussion by beam41** [kaogeek/kaogeek-discord-bot#24] [Ideas]
จะได้ติดตามง่ายๆเวลามี activity อะไรบน github
https://gist.github.com/jagrosh/5b1761213e33fc5b54ec7f6379034a22

**Comment by damp11113** [comment]
อันนี้ก็ดีอยู่

**Comment by LagSeeN** [comment]
@beam41 @damp11113 เพิ่มเรียบร้อยแล้วครับ
